### ISSUE TITLE
Fix or disable search feature

### ISSUE THREAD

**Issue by pcardune** [bootstrapworld/codemirror-blocks#485]
Depends on #482 

It doesn't have any tests and is currently broken. I know we'll need to have search functionality eventually, but is it necessary for the alpha release? If not, we should just disable it for now. AFAIK, the only way to access it is via a keyboard shortcut. Leaving this as a separate issue from #390 since that's just about code quality.
